src/lib/y2storage/dasd.rb

```diff
@@ -0,0 +1,50 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/storage_class_wrapper"
+require "y2storage/partitionable"
+
+module Y2Storage
+  # A DASD (direct-access storage device), typically used in mainframes
```

Comment:
@jreidinger Yes, that's better than nothing... but that's not the authoritative source that was used to write the class (unfortunately, there is no such link), so I see no reason to link to it in the class documentation. 

That link is simply the best you found with a google search today. Maybe tomorrow it will be dead or there would be a better source of documentation. If there is no concrete document we used as a guide, is probably up to the poor developer to search for "DASD direct-access storage device" in Internet. Sad but true, but pretending we extracted the information from a concrete place will not help much.